src/framework/ui/select/selectOption.component.tsx

```diff
@@ -34,6 +34,8 @@ export interface SelectOptionType {
   textStyle?: TextStyle;
   disabled?: boolean;
   items?: SelectOptionType[];
+
+  [key: string]: any;
```

Comment:
?

Reply:
This will override props, so that a component will accept any props without ts complains